app/vmui/packages/vmui/src/pages/ExploreAlerts/ExploreRules.tsx

```diff
@@ -68,7 +65,7 @@ const ExploreRules: FC = () => {
           groupId={groupId}
           id={ruleId}
           mode={ruleId !== "" ? "rule" : "alert"}
```

Comment:
This can be simplified to `mode={ruleId ? "rule" : "alert"}` if `ruleId` is always a string and `""` is the only value that should result in `"alert"`.

If `ruleId` can also be `null` / `undefined` / `0` and these cases should still map to `"rule"`, then the explicit `ruleId !== ""` check is correct and should be kept.

The same applies to other places where values are compared to an empty string.